### PR TITLE
Analytic user guides

### PR DESCRIPTION
**Pull request by matzew**
@corinnekrych @danielpassos please review

### CODE REVIEW

**Comment by danielpassos** [comment]
lgtm

File: docs/unifiedpush/aerogear-push-ios/iOS-app.asciidoc [hunk 1]
```diff
@@ -143,6 +143,36 @@ The _success_ callback is invoked, when the _AeroGear UnifiedPush Server_ was ab
...
+- (BOOL)application:(UIApplication *)application
+   didFinishLaunchingWithOptions:(NSDictionary *)launchOptions {
+       ...
+       // Send metrics when app is launched due to push notification
+       [AGPushAnalytics sendMetricsWhenAppLaunched:launchOptions];
+       ...
+}
+----
+
+If the application is launched using a push notification the _sendMetricsWhenAppLaunched_ static function needs to be invoked within the _didFinishLaunchingWithOptions_ method. when the app is brought from background to foreground due to a push notification, the following line needs to be added to the _didReceiveRemoteNotification_ method:
```
Comment (corinnekrych) [inline docs/unifiedpush/aerogear-push-ios/iOS-app.asciidoc]: /. when the app is brought/. When the app is brought/s
uppercase 'W'

**Comment by corinnekrych** [comment]
@matzew I'm fine with it. +1

I've just noticed that we should have a different ticket to migrate the doc to Swift code snippet + use HelloWorld https://github.com/jboss-mobile/unified-push-helloworld instead of https://github.com/aerogear/aerogear-push-ios-demo (this repo should be deprecated). 
https://issues.jboss.org/browse/AGIOS-452

**Comment by matzew** [comment]
@corinnekrych ah - good point! can you create those tickets for AGPUSH (1.1.1 fixed version) ?

**Pull request merged**